TU20Bot/Configuration/Controllers/AuthenticationController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Security.Cryptography;
+using System.Text;
+using EmbedIO;
+using EmbedIO.Routing;
+using EmbedIO.WebApi;
+using Jose;
+
+namespace TU20Bot.Configuration.Controllers {
+    public class AuthenticationController: ServerController {
+        public const int HASH_SIZE = 24;
+        private const int ITERATIONS = 1000000;
+        private static readonly string tempSalt = Environment.GetEnvironmentVariable("TEMPSALT");
+        private static readonly string tempSalt_default = "TEMPSALT";
+
+        [Route(HttpVerbs.Post, "/login")]
+        public string Login([QueryField] string password) {
+
+            var stringHash = hashPassword(password);
+            if(stringHash.Equals(this.server.config.consolePassword))
```

Comment:
```suggestion
            if (stringHash.Equals(this.server.config.consolePassword)) {
```

Sorry about style stuff. If you don't want to commit this feel free to resolve. When I see a multiple line statement connected to a single line if like this my brain gets a bit fuzzy.